Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/SubscriptionTests.cs

```diff
@@ -0,0 +1,153 @@
+﻿using FluentAssertions;
+using Kentico.Kontent.Management.Models.Shared;
+using Kentico.Kontent.Management.Models.Subscription;
+using Kentico.Kontent.Management.Tests.Unit.Base;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Kentico.Kontent.Management.Tests.Unit.ManagementClientTests
+{
+    public class SubscriptionTests : IClassFixture<FileSystemFixture>
+    {
+        private FileSystemFixture _fileSystemFixture;
+        private readonly string _subscriptionEndpoint; 
+
+        public SubscriptionTests(FileSystemFixture fileSystemFixture)
+        {
+            _fileSystemFixture = fileSystemFixture;
+            _fileSystemFixture.SetSubFolder("Subscription");
+            _subscriptionEndpoint = $"{_fileSystemFixture.Endpoint}/subscriptions/{_fileSystemFixture.SUBCRIPTION_ID}";
+        }
+
+        [Fact]
+        public async Task ListSubscriptionProjects_ListsSubscriptionProjects()
+        {
+            var expected = _fileSystemFixture.GetItemsOfExpectedListingResponse<SubscriptionProjectModel>("Projects.json");
+            var client = _fileSystemFixture.CreateMockClientWithResponseAndUrl("Projects.json", $"{_subscriptionEndpoint}/projects");
```

Comment:
I extended `FileSystemFixture` class so now we can tests URL directly in the unit tests and we don't need separate tests for `EndpointUrlBuilder`

Reply:
Cool, do we want to submit an issue to rewrite current tests?

Is this a good idea to connect these - I feel it does break the separation of concerns a bit.

Reply:
Let's keep this testing - `UrlBulderTests`.